formidable/forms/__init__.py

```diff
@@ -113,10 +114,34 @@ class MyForm(FormidableForm):
     """
 
     @classmethod
-    def to_formidable(cls, label, description=u''):
-        form = Formidable.objects.create(label=label, description=description)
+    def to_formidable(cls, label=None, description=None, instance=None):
```

Comment:
Yeah true. The docstring from the class was not viewable from the PR diff.
Looks good to me here. :wink: